cypress/integration/spawn/volume.ts

```diff
@@ -154,12 +154,6 @@ describe("Navigating to Spawn Volume page", () => {
     cy.contains(errorBannerCopy).should("exist");
   });
 
-  it("Switching tabs should clear the error toast.", () => {
```

Comment:
This behavior existed when we we used banners. But since the toasts are persistent in a context they exist throughout the app and disappear on their own over time. This test always failed initially and would only pass when it was automatically restarted by cypress. (which refreshed the page and made the toast go away)